packages/mdc-top-app-bar/foundation.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {strings} from './constants';
+import MDCFoundation from '@material/base/foundation';
+
+/**
+ * @extends {MDCFoundation<!MDCTopAppBarFoundation>}
+ * @final
+ */
+export default class MDCTopAppBarFoundation extends MDCFoundation {
+  /** @return enum {string} */
+  static get strings() {
+    return strings;
+  }
+
+  /**
+   * {@see MDCTopAppBarAdapter} for typing information on parameters and return
+   * types.
+   * @return {!MDCTopAppBarAdapter}
+   */
+  static get defaultAdapter() {
+    return /** @type {!MDCTopAppBarAdapter} */ {
+      hasClass: (/* className: string */) => /* boolean */ false,
```

Comment:
Pretty sure this should be `{}` on the RHS, as in, it's an empty function (not a return value).

I realize we seem to be 50/50 on this throughout the repo currently but I'm inclined to think it's a mistake.

Reply:
It was copied from the toolbar.